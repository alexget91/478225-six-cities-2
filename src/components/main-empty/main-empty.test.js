import React from "react";
import renderer from "react-test-renderer";
import MainEmpty from "./main-empty";

it(`Main empty correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<MainEmpty
      cityName={`1`}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
