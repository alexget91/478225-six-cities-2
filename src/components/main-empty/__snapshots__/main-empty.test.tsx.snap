// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main empty correctly renders after relaunch 1`] = `
Array [
  <section
    className="cities__no-places"
  >
    <div
      className="cities__status-wrapper tabs__content"
    >
      <b
        className="cities__status"
      >
        No places to stay available
      </b>
      <p
        className="cities__status-description"
      >
        We could not find any property availbale at the moment in 
        1
      </p>
    </div>
  </section>,
  <div
    className="cities__right-section"
  />,
]
`;
